backend/src/services/release.ts

```diff
@@ -56,8 +56,7 @@ async function validateRelease(user: UserInterface, model: ModelDoc, release: Re
     const fileNames: Array<string> = []
 
     for (const fileId of release.fileIds) {
-      let file: Document<unknown, any, FileInterface> &
-        Omit<FileInterface & Required<{ _id: Schema.Types.ObjectId }>, never>
+      let file: FileInterface | (undefined & Omit<FileInterface & Required<{ _id: Schema.Types.ObjectId }>, never>)
```

Comment:
I'm not sure what's happened here again. `let file: FileInterfaceDoc | undefined` should be the correct type

Reply:
Hmmm... fixed